src/modules/commander/HealthAndArmingChecks/checks/estimatorCheck.cpp

```diff
@@ -669,6 +671,32 @@ void EstimatorChecks::gpsNoLongerValid(const Context &context, Report &reporter)
 	}
 }
 
+void EstimatorChecks::lowPositionAccuracy(const Context &context, Report &reporter,
+		const vehicle_local_position_s &lpos) const
+{
+	const bool local_position_valid_but_low_accuracy = !reporter.failsafeFlags().local_position_invalid
+			&& (_param_com_low_eph.get() > FLT_EPSILON && lpos.eph > _param_com_low_eph.get());
+
+	if (!reporter.failsafeFlags().local_position_accuracy_low && local_position_valid_but_low_accuracy) {
+
+		// only report if armed
+		if (context.isArmed()) {
+			/* EVENT
+			 * @description Local position estimate valid but has low accuracy. Warn user.
+			 * This check can be configured via <param>COM_POS_LOW_EPH</param> parameter.
```

Comment:
If you add this, the GCS can hide the param hint for end-users:
```
<profile name="dev">
		 * This check can be configured via <param>COM_POS_LOW_EPH</param>
		 * </profile>
```

Reply:
Makes sense, 9c3173696d8100883c445892196ffb62ff440098